manager/integration/tests/test_engine_upgrade.py

```diff
@@ -1196,3 +1196,16 @@ def test_engine_live_upgrade_while_replica_concurrent_rebuild(client, # NOQA
     for replica in volume2.replicas:
         assert replica.image == engine_upgrade_image
         assert replica.currentImage == engine_upgrade_image
+
+@pytest.mark.skip(reason="TODO")  # NOQA
+def test_engine_crash_during_live_upgrade():
+    """
+    1. Create and attach a volume.
+    2. Deploy an extra engine image.
+    3. Send live upgrade request then immediately delete the related engine
+       manager pod/engine process (The new replicas are not in active in this
+       case).
+    4. Verify the volume will detach then reattach automatically.
+    5. Verify the upgrade is done during the reattachment.
+       (It actually becomes offline upgrade.)
```

Comment:
Besides testing the volume behavior alone, it would be beneficial to include a scenario where a workload is using that volume.

Reply:
Updated!